ngraph/core/include/ngraph/op/matrix_nms.hpp

```diff
@@ -0,0 +1,138 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "ngraph/op/util/nms_base.hpp"
+
+namespace ngraph
+{
+    namespace op
+    {
+        namespace v8
+        {
+            /// \brief MatrixNms operation
+            ///
+            class NGRAPH_API MatrixNms : public util::NmsBase
```

Comment:
Do we have a specification for this operation?

Reply:
Spec for the 2 Operations:
https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/sort/MatrixNMS_8.md
https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/sort/MulticlassNMS_8.md